src/custom/pages/Account/Tokens/TokenSingle.tsx

```diff
@@ -0,0 +1,24 @@
+import { Content } from 'components/Page'
+import { Wrapper, AccountPageWrapper } from '../styled'
+import { AccountMenu } from '../Menu'
+import { useParams } from 'react-router-dom'
+
+type RouterParams = {
+  address: string
+}
+
+export default function Token() {
+  const { address } = useParams<RouterParams>()
+
+  return (
+    <Wrapper>
+      <AccountMenu />
+      <AccountPageWrapper>
+        <Content>
+          <h2 id="tokens">Single token page</h2>
```

Comment:
all these text parts should be wrapped in `Trans`

Reply:
This page will probably change or be deleted so I wouldn't worry about it at this point